app/src/main/res/values/strings_activity_settings.xml

```diff
@@ -24,6 +24,22 @@
         <item>never</item>
     </string-array>
 
+    <string-array name="pref_playlists_last_played_interval_titles">
```

Comment:
If these intervals are going to be the same, we could just refactor them so that both settings use the same (generic) arrays.

Reply:
I d push this further, to have a control allowing users to freely compose the time range, i.e NN {day|week|month}.

The seekarc control used now in sleeptimer can be a good start.

To keep things separated, that is not the subject of this PR.